baseplate/core.py

```diff
@@ -39,15 +39,12 @@ class SpanObserver(object):
 
     def on_start(self):
         """Do something when the observed span is started."""
-        pass
```

Comment:
Why was this `pass` removed?  That seems _less_ intuitive.

Reply:
pylint has a lint for "unnecessary pass" when something like a docstring has already made it a syntactically valid indenty block.